lib/client.js

```diff
@@ -0,0 +1,130 @@
+const http = require('http'),
+    https = require('https'),
+    SolrError = require('./error/solr-error'),
+    JSONbig = require('json-bigint');
+
```

Comment:
extracted part of solr.js so that it could be reused elsewhere (in my cases, in `schemaHelper`)